termino/termino.py

```diff
@@ -156,21 +184,23 @@ async def settings(self, ctx: commands.Context):
             f"Shutdown message: {config['shutdown_message']}\n"
             f"Shutdown confirmation: {config['confirm_shutdown']}\n\n"
             f"Restart message: {config['restart_message']}\n"
```

Comment:
I think here, it's going to get confusing between the different settings (restart message and restarted message sound the same). It would be nice to call this setting a different name in this command, or add a note at the bottom. I don't have a preference.